examples/code/maps-and-hash-tables/map_vs_hash/run_map_vs_hash.sh

```diff
@@ -1,4 +1,5 @@
   $ jbuilder build map_vs_hash.exe
+  Done: 3/5 (jobs: 1)
                   
Done: 74/77 (jobs: 1)
                     
Done: 75/77 (jobs: 1)
                     
Done: 76/77 (jobs: 1)
                     

```

Comment:
Hm, we do need a quieter mode in jbuilder -- this progress bar shouldn't be captured in the output.